File: add/codesnippet/VisualBasic/p-system.datetimeoffset._15_2.vb
      Dim dto As DateTimeOffset
      Dim ctr As Integer
      Dim ms As Integer
      Do
         dto = DateTimeOffset.Now
         If dto.Millisecond <> ms Then
            ms = dto.Millisecond
            Console.WriteLine("{0}:{1:d3} ms. {2}", _
                              dto.ToString("M/d/yyyy h:mm:ss"), _
                              ms, dto.ToString("zzz"))
            ctr += 1
         End If
      Loop While ctr < 100

File: add/codesnippet/CSharp/p-system.datetimeoffset._15_2.cs
      DateTimeOffset dto;
      int ctr = 0;
      int ms = 0;
      do {
         dto = DateTimeOffset.Now;
         if (dto.Millisecond != ms)
         {
            ms = dto.Millisecond;
            Console.WriteLine("{0}:{1:d3} ms. {2}", 
                              dto.ToString("M/d/yyyy h:mm:ss"), 
                              ms, dto.ToString("zzz"));
            ctr++;
         }
      } while (ctr < 100);